Roslyn.sln

```diff
@@ -523,21 +523,13 @@ Global
 		{5F8D2414-064A-4B3A-9B42-8E2A04246BE5}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{5F8D2414-064A-4B3A-9B42-8E2A04246BE5}.Release|Any CPU.Build.0 = Release|Any CPU
 		{02459936-CD2C-4F61-B671-5C518F2A3DDC}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{02459936-CD2C-4F61-B671-5C518F2A3DDC}.Debug|Any CPU.Build.0 = Debug|x64
```

Comment:
Don't know what you did here but all of these changes are why your microbuild PR is failing.  

``` txt
13:39:41   The project "CompilersBoundTreeGenerator" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "CSharpErrorFactsGenerator" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "CSharpSyntaxGenerator" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "VisualBasicSyntaxGenerator" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "VisualBasicErrorFactsGenerator" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "CsiCore" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "VbiCore" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "DeployCoreClrTestRuntime" is not selected for building in solution configuration "Release|Any CPU".
13:39:41   The project "DeployCompilerGeneratorToolsRuntime" is not selected for building in solution configuration "Release|Any CPU".
13:39:42 Project "D:\j\workspace\microbuild_pr---d013602c\Roslyn.sln" (2) is building "D:\j\workspace\microbuild_pr---d013602c\src\Compilers\Core\CodeAnalysisTest\CodeAnalysisTest.csproj" (3) on node 1 (default targets).
```